setup.py

```diff
@@ -28,7 +28,7 @@
     keywords="api graphql protocol rest",
     packages=find_packages(exclude=["tests"]),
     install_requires=required_packages,
-    tests_require=["pytest>=2.7.3"],
+    tests_require=["pytest>=4.3.1"],
```

Comment:
I don't see in the CR where you're changing or adding tests that use any new features of pytest; why did you bump the version here?